app/src/main/java/com/djinc/edumotive/screens/PartDetails.kt

```diff
@@ -92,95 +90,131 @@ fun Details(
+                                val params = Bundle()
+                                params.putString("type", modelType)
+                                params.putString("id", modelId)
+                                intent.putExtras(params)
+                                context.startActivity(intent)
+                            }
                     ) {
-                        Icon(
-                                painter = painterResource(id = R.drawable.ic_augmented_reality),
-                                contentDescription = "See in Augmented Reality",
-                                tint = PinkPrimary
-                        )
-                        Text(
-                                text = "Open in AR",
-                                color = PinkPrimary,
-                                fontSize = 15.sp,
-                                modifier = Modifier.padding(top = 2.dp)
-                        )
+                        Row(
+                                horizontalArrangement = Arrangement.spacedBy(4.dp),
+                                verticalAlignment = Alignment.CenterVertically,
+                                modifier = Modifier.padding(horizontal = 8.dp, vertical = 4.dp)
+                        ) {
+                            Icon(
+                                    painter = painterResource(id = R.drawable.ic_augmented_reality),
+                                    contentDescription = "See in Augmented Reality",
+                                    tint = PinkPrimary
+                            )
+                            Text(
+                                    text = "Open in AR",
+                                    color = PinkPrimary,
+                                    fontSize = 15.sp,
+                                    modifier = Modifier.padding(top = 2.dp)
+                            )
+                        }
                     }
                 }
             }
-        }
-        item {
-            Box(
-                    contentAlignment = Alignment.Center, modifier = Modifier
-                    .fillMaxWidth(1f)
-                    .aspectRatio(1F)
-                    .background(Background, RoundedCornerShape(8.dp))
-            ) {
-                AsyncImage(imageUrl = imageUrl, imageName = "TODO")
-            }
-        }
-        item {
-            Text(text = "Informatie", style = MaterialTheme.typography.h4)
-            Text(
-                    text = description,
-                    style = MaterialTheme.typography.body2,
-            )
-        }
-        if (!models.isEmpty()) {
             item {
-                ScreenTitle(title = "Bijbehorende onderdelen", spacerHeight = 0, windowSize = windowSize)
+                Box(
+                        contentAlignment = Alignment.Center, modifier = Modifier
+                        .fillMaxWidth(1f)
+                        .aspectRatio(1F)
```

Comment:
Capital F?

Reply:
spelling mistake but makes no difference to the way it works